.github/workflows/bump-version.yml

```
@@ -0,0 +1,91 @@
+              )
+
+          with open(path, "w") as f:
+              f.write(content)
+
+          print(f"Firmware bumped to {major}.{minor}.{patch} (protocol {proto})")
+          PYEOF
+
+      - name: Commit and push
+        run: |
+          VERSION=$(python3 -c "
+import re
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)